src/monitoring_service/schema.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE blockchain (
+    chain_id                        INTEGER,
+    receiver                        CHAR(42),
+    token_network_registry_address  CHAR(42),
+    monitor_contract_address        CHAR(42),
+    latest_known_block              INT
+);
+INSERT INTO blockchain VALUES (
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+    NULL
+);
+
+CREATE TABLE token_network (
+    address                 CHAR(42) PRIMARY KEY
+);
+
+
+CREATE TABLE channel (
+    token_network_address   CHAR(42) NOT NULL,
+    identifier              HEX_INT  NOT NULL,
+    participant1            CHAR(42) NOT NULL,
+    participant2            CHAR(42) NOT NULL,
+    settle_timeout          HEX_INT  NOT NULL,
+    -- see raiden_contracts.constants.ChannelState for value meaning
+    state                   INT NOT NULL CHECK (state >= 0 AND state <= 4),
```

Comment:
Not for this PR, just a thought: atm we don't use the state, maybe we can remove it all together

Reply:
I would be surprised if we won't need this for some validation or idempotence guarantees.